.travis.yml

```diff
@@ -9,8 +9,8 @@ cache:
 matrix:
     fast_finish: true
     include:
-        - php: 7.0
-        - php: 7.0
+        - php: '7.0'
```

Comment:
Still means that under the hood they didn't care if it was `5.6` or `'5.6'`. And that's still the case I think, they must be changing it because:

- they **had** to use this input as string for node case (at least)
- as they have to, it's better to be consistent and do that everywhere

I don't think they will remove the support for this "error" without a warning though, otherwise over half of PHP builds will simply break ^^